ndex2/client.py

```diff
@@ -354,6 +383,19 @@ def save_new_network(self, cx, visibility=None):
     def save_cx_stream_as_new_network(self, cx_stream, visibility=None):
         """
         Create a new network from a CX stream.
+	
+	.. code-block:: python
+	
+		import ndex2.client
+		my_account="your account"
+		my_password="your password"
+		my_ndex=ndex2.client.Ndex2("http://public.ndexbio.org", my_account, my_password)
+
+		node_name = 'INSERT NODE NAME HERE'
+		network_uuid = 'INSERT NETWORK UUID HERE'
+		query_result_cx_stream = my_ndex.get_neighborhood_as_cx_stream(network_uuid, node_name)
```

Comment:
Instead of making the user find a network they can run a neighborhood query on just directly create a network with this fragment of code (you'll have to move the import to the top.

```
from ndex2.nice_cx_network import NiceCXNetwork

my_net = NiceCXNetwork()
my_net.create_node('testnode1')
my_net.set_name('testnetwork')
uri_subset = my_ndex.save_new_network(my_net.to_cx())
```